search_index.py

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/python
+# Copyright 2019 Gentoo Authors
+# Distributed under the terms of the GNU GPL version 2 or later
+import json
+import os.path
+import sys
+import xml.etree.ElementTree as ET
+
+files = sys.argv[1:]
+documents = []
+url_root = 'https://devmanual.gentoo.org/'
+
+def stringify_node(parent):
+    if parent.text:
+        text = parent.text.strip().replace('\n', ' ')
```

Comment:
shouldn't this be the other way around maybe?

Reply:
Let me re-examine the behavior of ElementTree on empty tags (such as `<d/>`) and get back to you.